src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/Tizen/IVideoOutput.tizen.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public interface IVideoOutput
```

Comment:
Since we have a lot of nested classes and interfaces here, can you rename the File Name? Or even move other members to separated files?

Reply:
Yes... I was just afraid of having too many Tizen files.. I would rename the file name.